packages/grafana-ui/src/components/GraphNG/events.ts

```diff
@@ -0,0 +1,19 @@
+import { BusEventBase, BusEventWithPayload } from '@grafana/data';
+
+/** @alpha */
+export interface SetGraphCursorEventPayload {
+  point: { time: number };
+}
+
+/**
+ * Event used for one-way communication to set cursor at all time series panel at given time
+ * @alpha
+ */
+export class SetGraphNGCursorEvent extends BusEventWithPayload<SetGraphCursorEventPayload> {
+  static type = 'graph-ng-set-cursor';
```

Comment:
Why do we need a new event type?

Reply:
Yeah, actually we don't a new event. I could re-use `LegacyGraphHoverEvent` making it super duper simple like [this](https://github.com/grafana/grafana/compare/ifrost/logs-time-series-sync?expand=1). It just felt weird to dispatch old graph legacy event from logs panel. Not sure what's the plan for the future - do we intend to remove the old graph and `LegacyGraphHoverEvent`?

Reply:
> do we intend to remove the old graph and LegacyGraphHoverEvent?

The naming here is super unfortunate, as It's not a legacy event, but event that was introduced to support **legacy visualization*.

What I suggest doing here is renaming the `LegacyGraphHoverEvent` instead of introducing a new event type.

Name suggestions:
- `DatapointHoverEvent`
- `VizHoverEvent`